### ISSUE TITLE
Running the example failed

### ISSUE THREAD

**Issue by SenthilPanneerselvam** [AgoraIO-Community/Agora-RN-Quickstart#72]
I tried running the project by downloading and followed the instruction but then it failed in the step:
npx react-native run-ios

The error message I see in the console is:
The following build commands failed:
        CompileC /Users/senthilkumarpanneerselvam/Library/Developer/Xcode/DerivedData/AgoraRNQuickstart-akjxwaguckfbkpfronshzhawybnf/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/React-Core.build/Objects-normal/x86_64/RCTCxxBridge.o /Users/senthilkumarpanneerselvam/Projects/Agora/Agora-RN-Quickstart-master/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm normal x86_64 objective-c++ com.apple.compilers.llvm.clang.1_0.compiler

**Comment by LichKing-2234** [comment]
Pls try the example which in the repository.

**Comment by SenthilPanneerselvam** [comment]
Yes I tried the example within the repository. Thats when I faced this issue. After some search found that this is a problem with xcode new version. Am in the process of downgrading the xcode. I will check and if that solves the issue I will close this.
